base/epics-base-musl.patch

```diff
@@ -0,0 +1,42 @@
+From 7c4a21eab44183a84f25ea234ce2fde8ad08c4ed Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?=C3=89rico=20Nogueira?= <erico.rolim@lnls.br>
+Date: Mon, 31 Jul 2023 11:36:11 -0300
+Subject: [PATCH] libCom: detect support for backtrace() with __has_include.
```

Comment:
I would be ok with naming the file "backport...". Then it's clear that if applying it fails, it can probably just be removed.

Reply:
Nice. [nanohttp_stream.patch](https://github.com/cnpem/epics-in-docker/blob/v0.4.0/base/nanohttp_stream.patch) can also follow this pattern now, since it was already merged.